run

```diff
@@ -493,7 +493,7 @@ lint() {
     pushd ${SCRIPT_DIR} > /dev/null
 
     echo "Linting Python"
-    run_command ruff $DIR_TO_RUN --ignore E712 || exit_code=1
+    run_command ruff check $DIR_TO_RUN --ignore E712 || exit_code=1
```

Comment:
@mocsharp were you seeing an error without `check`?

Reply:
https://github.com/nvidia-holoscan/holohub/actions/runs/9703101172/job/26780463857#step:4:7

```
Linting Python
[command] ruff . --ignore E712
error: `ruff <path>` has been removed. Use `ruff check <path>` instead.
```

Reply:
Which version of ruff are you using?
I guess adding check does not hurt for previous versions...

Reply:
Yea, looks fine to update this.

Reply:
@jjomier it's coming from GitHub actions. Doesn't look like we pin to any specific version in the run script so likely the latest.

Reply:
That makes sense - thanks!